pages.it/common/git-reset.md

```diff
@@ -0,0 +1,33 @@
+# git reset
+
+> Annulla commit o rimuovi modifiche dall'area di stage, reimpostando l'HEAD corrente su uno specifico stato.
+> Se viene fornito un percorso, il comando reset si interpreta come "rimuovi dall'area di stage"; se viene fornito l'hash di un commit o un ramo, si interpreta come "annulla commit".
+> Maggiori informazioni: <https://git-scm.com/docs/git-reset>.
+
+- Rimuovi tutto dall'area di stage:
+
+`git reset`
+
+- Rimuovi dall'area di stage uno o più file:
+
+`git reset {{percorso/a/uno/o/più/file}}`
```

Comment:
It's a long path. and it isn't clear, because you can put 2 files in the same path only using `/file{1,2}` :scream_cat: 
Do you want to talk about `a space-separated list of files`?